server/schema/mutation/signUp.js

```diff
@@ -0,0 +1,27 @@
+import {
+  GraphQLString,
+  GraphQLNonNull,
+  GraphQLObjectType,
+} from 'graphql';
+import userType from '../object-types/user-type';
+import * as UserController from '../../db/mongo/controllers/users';
+
+const signUpPayload = new GraphQLObjectType({
+  name: 'SignUpPayload',
+  fields: {
+    user: {type: userType},
+    token: {type: GraphQLString},
+  }
+});
+
+const signUp = {
+  args: {
+    name: {type: new GraphQLNonNull(GraphQLString)},
```

Comment:
So we require not only an email/password but a user name as well. Isn't it too much for a simple todo app? 😃 I'd suggest to make it optional at least.